packs/coder-2.21.3/values.yaml

```diff
@@ -0,0 +1,357 @@
+# coder -- Primary configuration for `coder server`.
+coder:
```

Comment:
The format for embedding the helm chart values in pack values.yaml should be like
```
charts:
  coder:
    # All coder charts values
```

For more info, check the [doc link](https://docs.spectrocloud.com/registries-and-packs/adding-add-on-packs/#helm-charts)